README.md

```diff
@@ -133,6 +133,30 @@ Add the following to your `config.php`:
     ),
 
 
+XMPP
```

Comment:
maybe specify prosody already here in the title?

Reply:
implemented.